lib/gap_intelligence/client/requestable.rb

```diff
@@ -14,7 +14,7 @@ def perform_request(method, path, options = {}, &block)
 
       response = connection.request(method, path, options, &block)
 
-      return RequestError.new(response.parsed['error']) if response.error
+      return RequestError.new(parse_error_message(response)) if response.error
```

Comment:
yea, we definitely need to check it or else it will always return a RequestError object.